docs/results.md

```diff
@@ -0,0 +1,376 @@
+```
+results/scale/PERF_SCALE_1572446292
+├── Scale_1572446292_1_600
+├── Scale_1572446292_2_700
+└── Scale_1572446292_3_800
+```
+
+### Iteration sub-directories
+The sub-directory created for each Scale test iteration contains directories and files selectively copied from the corresponding Performance results for that iteration.
+For example:
+```
+results/scale/PERF_SCALE_1572446292/Scale_1572446292_1_600
+├── 20191030T143909Z-20191030T151209Z.tar.gz
+├── current_tune_settings.json
+├── end_epoch
+├── master
+├── metric
+├── puppet-metrics-collector
+├── puppetserver.average.csv
+├── puppetserver.average.csv.html
+├── puppetserver.csv
+├── puppetserver.csv.html
+└── start_epoch
+```
+
+#### Host directories
+The first obvious difference between the Scale results and the Performance results is that the 'master' and 'metric' nodes are identified by their role rather than hostname.
+
+##### Master
+The 'master' directory contains the same files found in the corresponding Performance results directory.
+For example:
+```
+results/scale/PERF_SCALE_1572446292/Scale_1572446292_1_600/master
+├── atop_log_scale_1572446292_1_600_json.csv
+├── atop_log_scale_1572446292_1_600_json.detail.csv
+├── atop_log_scale_1572446292_1_600_json.detail.csv.html
+├── atop_log_scale_1572446292_1_600_json.log
+├── atop_log_scale_1572446292_1_600_json.log.tar.gz
+├── atop_log_scale_1572446292_1_600_json.summary.csv
+└── atop_log_scale_1572446292_1_600_json.summary.csv.html
+```
+
+These have been covered in the Performance [Performance/Host directories/Master](#master) section above.
+
+##### Metric
+The 'metric' directory contains the Gatling results found in the corresponding `root/gatling-puppet-load-test/simulation-runner/results` directory in the Performance results.
+For example:
+
+```
+results/scale/PERF_SCALE_1572446292/Scale_1572446292_1_600/metric
+├── PerfAutoScale-1572446523904.csv
+├── PerfAutoScale-1572446523904.csv.html
+├── group_perftestlarge-5163d.html
+├── index.html
+├── js
+├── req_perftestlarge---588ae.html
+├── req_perftestlarge---8453a.html
+├── req_perftestlarge---a1275.html
+├── req_perftestlarge---b4f51.html
+├── req_perftestlarge---db667.html
+├── req_perftestlarge---de2f0.html
+├── simulation.log
+└── style
+```
+
+These have been covered in the Performance [Performance/Host directories/Metric](#metric) section above.
+
+## Gatling JSON data
+The `stats.json` and `global_stats.json` files for each iteration are copied to the `json` directory and renamed based on the iteration naming pattern used for the iteration sub-directories.
+For example:
+```
+results/scale/PERF_SCALE_1572446292/json
+├── Scale_1572446292_1_600global_stats.json
+├── Scale_1572446292_1_600stats.json
+├── Scale_1572446292_2_700global_stats.json
+├── Scale_1572446292_2_700stats.json
+├── Scale_1572446292_3_800global_stats.json
+└── Scale_1572446292_3_800stats.json
+```
+
+## Logs
+The log files for the Scale test run are copied to the `log` directory.
+For example:
```

Comment:
I think you want a blank line before this so that it doesn't get folded into the previous paragraph when rendered.